creator/ingest_runs/data_generator/study_generator.py

```diff
@@ -1,162 +1,267 @@
 """
-A fake KF study generator that generates realistic studies
+A fake KF study generator that generates realistic Data Service studies
 Used to setup unit tests for ingest runs and related functionality
 
-1. Create (or read previously created) study files
-- Create a biological TSV file with minimal family, participant, specimen data
-- Create source and harmonized genomic files
-- Upload genomic files to study bucket on S3
-- Create S3 object manifest for source genomic files
-- Create the BIX Genomic Workflow Output Manifest
-
-2. Ingest study files into the KF Data Service (except the BIX manifest)
+1. (Optional) Delete all previously data on disk, delete study in Data Service
```

Comment:
`1. (Optional) Delete all previously generated data on disk, delete study in Data Service`